libraries/rush-lib/src/logic/base/BaseInstallManager.ts

```diff
@@ -293,11 +293,12 @@ export abstract class BaseInstallManager {
     if (this.rushConfiguration.isPnpm && experiments?.usePnpmSyncForInjectedDependencies) {
       const pnpmLockfilePath: string = subspace.getTempShrinkwrapFilename();
       const dotPnpmFolder: string = `${subspace.getSubspaceTempFolderPath()}/node_modules/.pnpm`;
+      const modulesFilePath: string = `${subspace.getSubspaceTempFolderPath()}/node_modules/.modules.yaml`;
```

Comment:
Ideally [pnpm-sync-lib](https://github.com/tiktok/pnpm-sync/blob/7a14a9595e1b66a7595a809acd9e7b943cfe0bc1/packages/pnpm-sync-lib/src/pnpmSyncPrepare.ts#L224) should export this as a constant.  But it's not essential.